src/metabase/driver/postgres.clj

```diff
@@ -626,12 +626,12 @@
     "inet"  :type/IPAddress
     nil))
 
-(defn- pkcs-12-key-value?
-  "If a value was uploaded for the SSL key, return whether it's using the PKCS-12 format."
+(defn- key-file-ext
   [ssl-key-value]
   (when ssl-key-value
-    (= (second (re-find secret/uploaded-base-64-prefix-pattern ssl-key-value))
-       "x-pkcs12")))
+    (case (second (re-find secret/uploaded-base-64-prefix-pattern ssl-key-value))
+      "x-pkcs12" ".p12"
```

Comment:
we could expand this to include `.pem` etc., but not sure if there's any benefit